<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KinectControllerClass</class>
 <widget class="QMainWindow" name="KinectControllerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KinectController</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>409</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="open_Kinect_Button">
         <property name="text">
          <string>开启Kinect相机</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="show_cur_pointcloud">
         <property name="text">
          <string>展示当前点云</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_18">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="save_cur_pointcloud">
           <property name="text">
            <string>保存当前点云</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="save_filename_lineEdit">
           <property name="text">
            <string>C:/Users/Dy/Desktop/</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="readPCD">
           <property name="text">
            <string>读点云图</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pcd_filename_lineEdit">
           <property name="text">
            <string>C:/Users/Dy/Desktop/</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>525</width>
      <height>556</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_26" stretch="1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,6">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>DK_ROI:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_18">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>DK_minX:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="DK_minX_lineEdit">
                 <property name="text">
                  <string>0.05</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_19">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>DK_maxX:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="DK_maxX_lineEdit">
                 <property name="text">
                  <string>0.2</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_20">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_21">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>DK_minY:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="DK_minY_lineEdit">
                 <property name="text">
                  <string>0.05</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_22">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>DK_maxY:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="DK_maxY_lineEdit">
                 <property name="text">
                  <string>0.2</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_24">
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>DK_minZ:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="DK_minZ_lineEdit">
                 <property name="text">
                  <string>0.05</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_25">
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>DK_maxZ:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="DK_maxZ_lineEdit">
                 <property name="text">
                  <string>0.2</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="Kinect_color_image">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>256</width>
             <height>144</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>256</width>
             <height>144</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;Agency FB&quot;;
border:2px groove gray;</string>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="text">
            <string>彩色图像区域</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Kinect_depth_image">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>256</width>
             <height>144</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>256</width>
             <height>144</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;Agency FB&quot;;
border:2px groove gray;</string>
           </property>
           <property name="text">
            <string>深度图像区域</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="MyQOpenglWidget" name="pointcloud_widget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>512</width>
           <height>288</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>512</width>
           <height>288</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyQOpenglWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>myqopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
